metabox/metabox/scenarios/config/environment.py

```diff
@@ -23,6 +23,7 @@
 from metabox.core.actions import Start
 from metabox.core.actions import Put
 from metabox.core.scenario import Scenario
+from metabox.core.utils import tag
```

Comment:
This is called here, but never used I think.

Reply:
The `tag` is the decorator that you commended above. Or am I misunderstanding your comment?

Reply:
Fixed in 7f2ac4a40